docs/tutorials/running-the-dal-as-a-weeklynet-baker-in-5-steps/run-dal-node.md

```diff
@@ -0,0 +1,22 @@
+
+## Step 4: run an Octez DAL node on Weeklynet
+
+```
+octez-dal-node run &>> "$HOME/octez-dal-node.log"
```

Comment:
When I do this in the Weeklynet Docker container, I get this error:
```
/bin/sh: syntax error: unexpected redirection
```

Reply:
I have to admit that I haven't tried with Docker. I guess this `&>>` construct is specific to bash and docker's shell is not bash.  Does it work with
```
octez-dal-node run >> "$HOME/octez-dal-node.log" 2>&1
```
?

Reply:
Yes, that works. The docker container must be on an older version of bash.

Reply:
Fixed in f489b0e